### DISCUSSION TITLE
全球最大離岸浮動式風機開始發電！挪威國家石油轉型里程碑，台灣能嗎

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#903] [General]
能源危機在全球變遷下已經成為了不可忽視的問題 隨著石油短缺愈演越烈 許多靠石油維生的國家也漸漸正視這個問題 希望在不久的將來能取到一個新平衡

https://tw.news.yahoo.com/%E5%85%A8%E7%90%83%E6%9C%80%E5%A4%A7%E9%9B%A2%E5%B2%B8%E6%B5%AE%E5%8B%95%E5%BC%8F%E9%A2%A8%E6%A9%9F%E9%96%8B%E5%A7%8B%E7%99%BC%E9%9B%BB-%E6%8C%AA%E5%A8%81%E5%9C%8B%E5%AE%B6%E7%9F%B3%E6%B2%B9%E8%BD%89%E5%9E%8B%E9%87%8C%E7%A8%8B%E7%A2%91-%E5%8F%B0%E7%81%A3%E8%83%BD%E5%97%8E-013259233.html

**Comment by Kotori0501** [comment]
台灣大多仰賴進口、化石燃料，這樣空氣汙染會日益嚴重，也會對人體健康造成威脅，所以這時真的要好好提倡再生能源，比如：風力、太陽能、水力，這樣或許能達到日後淨零排放的目標，也可以減少空氣汙染。

**Comment by ruthchen1020** [comment]
台灣需要更多人投入這項產業，一起去規劃和發展綠色能源。希望綠色能源可以真的綠色，而不是消耗了很多的的地球資源，卻只產出比少少的能量，還造成無法循環使用大型垃圾。

**Comment by evanchou1204** [comment]
感覺以台灣目前的技術和條件，要跟上挪威還很困難，政府真的該好好重視綠色能源的議題。

**Comment by CPCpyt** [comment]
我覺得台灣要實行很有困難，畢竟土地沒那麼大，
沿海發電很站占空間而且成本巨大，也會影響到沿海居民的環境和收入。

**Comment by StevenWanglolz** [comment]
感覺台灣還離得遠呢! 先著眼於現今面臨的問題比較重要吧

**Comment by swerfg03joe** [comment]
這類型的文章大多報喜不報憂，就算台灣周圍真的有優質的風場，但台灣要甚麼時候才能將所有風機安裝完成，一年的故障率有多少，平均一支的維修費有多少，這些國外廠商通常就是看準了售後的商機才會大舉投資沒什麼風電技術的台灣，因為如果政策執行下去了卻沒有風電的替代方案我們只能任人宰割，還有據我所知台灣電網其實並不是特別完善，常常在用電高峰期就會跳開，先做好基礎建設才是重點

**Comment by ru0372572** [comment]
循環經濟很顯然的是對未來比較好的方法，但現在的主流經濟還是線性經濟，就是指開採到使用後直接廢棄，無法回收再使用。感覺道理大家都懂，但價格跟成本又是另一個考慮的重點。
